smoketest/twiliovoiceverification_test.go

```diff
@@ -74,7 +75,11 @@ func TestTwilioVoiceVerification(t *testing.T) {
 		return -1
 	}, msg.Body())
 
-	code, _ := strconv.Atoi(codeStr)
+	// Since verification code is said twice during one Twilio message
+	c := splitCode(codeStr, 6)
```

Comment:
We can simplify this using slice notation, we should probably assert on the length though.

Something like:
```go
assert.Len(t, codeStr, 12)
code, _ := strconv.Atoi(codeStr[:6])
```